sqld/src/admin_api.rs

```diff
@@ -1,25 +1,37 @@
 use anyhow::Context as _;
-use axum::{extract::State, Json};
-use serde::Deserialize;
-use std::net::SocketAddr;
+use axum::extract::{BodyStream, Path, State};
+use axum::Json;
+use futures::TryStreamExt;
```

Comment:
I would avoid using futures crate and use tokio-stream instead

Reply:
Why is that?